### ISSUE TITLE
error build.sh please helpe

### ISSUE THREAD

**Issue by M1n4t01sb4d** [dougy147/mcbash#21]
I already made the changes suggested in the other topics...

make and docker error 

─$ sudo docker-compose up -d
Building mcbash
Sending build context to Docker daemon   3.67MB
Step 1/5 : FROM alpine:latest as runner
 ---> 05455a08881e
Step 2/5 : RUN apk update && apk add --no-cache     gcc make bash ncurses curl wget
 ---> Using cache
 ---> a64b9fdebac4
Step 3/5 : WORKDIR /mcbash
 ---> Using cache
 ---> 821aa4c6a51f
Step 4/5 : COPY . .
 ---> fb5158afedef
Step 5/5 : RUN make && make install
 ---> Running in c0c3601489e1
mkdir -p /usr/local/bin
sh build.sh
: not foundine 2: 
build.sh: set: line 3: illegal option -                                                    
make: * [Makefile:14: install] Error 2                                                   
The command '/bin/sh -c make && make install' returned a non-zero code: 2                  
ERROR: Service 'mcbash' failed to build : Build failed

**Comment by dougy147** [comment]
I cannot reproduce on my side (`docker-compose` version 2.24).
Just to be sure, did you exactly follow this: 

```sh
git clone https://github.com/dougy147/mcbash
cd ./mcbash
docker-compose up -d # Avoid using sudo if you can.
```

If yes, try commenting out line 3 in `build.sh` ( `set -xe` -> `#set -xe`), and launch `docker-compose up -d` again.
Might be similar to [this](https://github.com/linuxserver/docker-baseimage-guacgui/issues/27) in another repo.

**Comment by dougy147** [comment]
This issue is solved by commit 629008f.
